lib/tasks/feed.rake

```diff
@@ -0,0 +1,27 @@
+require_relative '../../../app/workers/feeds/all_worker'
```

Comment:
this require statement seems to be breaking CircleCI:
```
rake aborted!
LoadError: cannot load such file -- /home/circleci/app/workers/feeds/all_worker
....
```

check the path is correct - it might be helpful to ssh into CircleCi and run a rails console there, where you can issue some `require` statements and debug the issue